### PR TITLE
Create journal-of-international-economic-law.csl

### PR DESCRIPTION
**Pull request by cathbrend**

### CODE REVIEW

**Comment by cathbrend** [comment]
I have modified my file, but I don't understand how I can copy the new file...

File: journal-of-international-economic-law.csl [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+  <macro name="author-note">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never"/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="internet-location">
+    <choose>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: no need for the choose-if here, that's implicit in `<text variable="URL".../>`. Please remove.

**Comment by rmzelle** [comment]
Looks fine to me. @adam3smith?

File: journal-of-international-economic-law.csl [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+  <macro name="issued-long">
+    <text variable="volume"/>
+    <date delimiter=" " variable="issued">
+      <date-part name="day"/>
+      <date-part name="month" form="long"/>
+      <date-part name="year" form="long"/>
+    </date>
+  </macro>
+  <macro name="edition">
+    <number variable="edition" form="ordinal" text-case="lowercase" suffix=" "/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: lowercase doesn't do anything here (it's also the default in English where it _does_ do anything). Remove, please.

File: journal-of-international-economic-law.csl [hunk 3]
```diff
@@ -0,0 +1,405 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="en-US">
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: you'll want to use `en-GB` as the default locale. Reason will become clear below.

File: journal-of-international-economic-law.csl [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <issn>1369-3034</issn>
+    <summary>According to the style sheet of the Journal of International Economic Law</summary>
+    <updated>2016-06-21T14:55:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: remove once locale changed to en-GB (which does this automatically).

File: journal-of-international-economic-law.csl [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+        <text value="4"/>
+      </else-if>
+      <else>
+        <text value="5"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="render-chapter">
+    <group>
+      <text variable="title" strip-periods="true" quotes="false" prefix="‘" suffix="’, "/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: don't hard-code quotes. Instead use `quotes="true"` with en-GB locale, those default to single quotes.

File: journal-of-international-economic-law.csl [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+        <text variable="genre"/>
+        <text variable="number"/>
+      </group>
+    </group>
+    <text macro="publisher-place-year"/>
+    <text macro="internet-location"/>
+  </macro>
+  <macro name="render-thesis">
+    <group delimiter=" ">
+      <text variable="title" font-style="normal" prefix="‘" suffix="’"/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: same here.

File: journal-of-international-economic-law.csl [hunk 7]
```diff
@@ -0,0 +1,405 @@
...
+        <text term="available at"/>
+        <text variable="publisher" suffix=","/>
+        <text variable="publisher-place" suffix=", "/>
+        <date date-parts="year" form="text" variable="issued"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="render-article-newspaper">
+    <group delimiter=", ">
+      <text variable="title" quotes="false" prefix="‘" suffix="’"/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: and here.

File: journal-of-international-economic-law.csl [hunk 8]
```diff
@@ -0,0 +1,405 @@
...
+      <text macro="container-title" font-style="italic"/>
+      <text macro="issued-long" suffix=","/>
+      <text variable="page-first"/>
+    </group>
+    <text macro="internet-location"/>
+  </macro>
+  <macro name="render-webpage">
+    <group delimiter=" " suffix=",">
+      <text variable="title" quotes="false" prefix="‘" suffix="’"/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: ditto

File: journal-of-international-economic-law.csl [hunk 9]
```diff
@@ -0,0 +1,405 @@
...
+          <date-part name="year"/>
+        </date>
+      </group>
+    </group>
+  </macro>
+  <macro name="render-bill">
+    <group delimiter=" ">
+      <group delimiter=", ">
+        <group>
+          <text variable="title" font-style="normal"/>
```
Comment (adam3smith) [inline journal-of-international-economic-law.csl]: remove redundant `font-style="normal"`

**Comment by adam3smith** [comment]
These should all be pretty quick changes. Looks good otherwise. Any problems/question let us know.

**Comment by cathbrend** [comment]
I have modified the file, I hope it is ok now...

File: journal-of-international-economic-law.csl [hunk 10]
```diff
@@ -0,0 +1,400 @@
...
+        <text value="4"/>
+      </else-if>
+      <else>
+        <text value="5"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="render-chapter">
+    <group>
+      <text variable="title" strip-periods="true" quotes="true" prefix="‘" suffix="’, "/>
```
Comment (rmzelle) [inline journal-of-international-economic-law.csl]: @cathbrend, if you use `quotes="true"`, quotation marks are added automatically, so you can remove them from the prefix/suffix attributes.

File: journal-of-international-economic-law.csl [hunk 11]
```diff
@@ -0,0 +1,400 @@
...
+      <text macro="editor" strip-periods="true" suffix=", "/>
+    </group>
+    <text macro="container-title" font-style="italic"/>
+    <text macro="edition" prefix=", "/>
+    <text macro="publisher-place-year"/>
+    <text variable="page" prefix=" "/>
+  </macro>
+  <macro name="render-article-journal">
+    <group delimiter=" ">
+      <text variable="title" quotes="true" prefix="‘" suffix="’,"/>
```
Comment (rmzelle) [inline journal-of-international-economic-law.csl]: Same here. (as above: https://github.com/citation-style-language/styles/pull/2122/files#r68236788)

File: journal-of-international-economic-law.csl [hunk 12]
```diff
@@ -0,0 +1,400 @@
...
+        <text term="available at"/>
+        <text variable="publisher" suffix=","/>
+        <text variable="publisher-place" suffix=", "/>
+        <date date-parts="year" form="text" variable="issued"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="render-article-newspaper">
+    <group delimiter=", ">
+      <text variable="title" quotes="true" prefix="‘" suffix="’"/>
```
Comment (rmzelle) [inline journal-of-international-economic-law.csl]: Idem.

File: journal-of-international-economic-law.csl [hunk 13]
```diff
@@ -0,0 +1,400 @@
...
+      <text macro="container-title" font-style="italic"/>
+      <text macro="issued-long" suffix=","/>
+      <text variable="page-first"/>
+    </group>
+    <text macro="internet-location"/>
+  </macro>
+  <macro name="render-webpage">
+    <group delimiter=" " suffix=",">
+      <text variable="title" quotes="true" prefix="‘" suffix="’"/>
```
Comment (rmzelle) [inline journal-of-international-economic-law.csl]: Idem.

File: journal-of-international-economic-law.csl [hunk 14]
```diff
@@ -0,0 +1,400 @@
...
+        <text variable="genre"/>
+        <text variable="number"/>
+      </group>
+    </group>
+    <text macro="publisher-place-year"/>
+    <text macro="internet-location"/>
+  </macro>
+  <macro name="render-thesis">
+    <group delimiter=" ">
+      <text variable="title" prefix="‘" suffix="’"/>
```
Comment (rmzelle) [inline journal-of-international-economic-law.csl]: You'd want to use `quotes="true"` here too.

**Comment by cathbrend** [comment]
Done. Thanks.

**Comment by rmzelle** [comment]
Great!

**Pull request merged**